### ISSUE TITLE
Plans for implementing SSIM loss

### ISSUE THREAD

**Issue by SomritaRPE** [JuliaImages/Images.jl#1023]
Hi All

I am unable to find any loss function in Julia corresponding to SSIM .Will there be any upgradation regarding this?
I've used it many time is the TensorFlow platform but there's no equivalent (that I could find) in Julia Images .It is very important loss for image to image translations, thus it would be grate if it becomes available here.

Best regards

Somrita

**Comment by ashwani-rathee** [comment]
Hey, I think you are mentioning this: https://github.com/JuliaImages/ImageQualityIndexes.jl/blob/master/src/ssim.jl

**Comment by ashwani-rathee** [comment]
This is being worked on here: https://github.com/FluxML/Flux.jl/pull/2178

**Comment by nikopj** [comment]
I've moved the work of the Flux pull request to a standalone repository here: https://github.com/nikopj/SSIMLoss.jl
SSIMLoss.jl is AD and GPU friendly + tested against ImageQualityIndexes.jl for correctness.
